app/src/main/res/values/themes.xml

```diff
@@ -52,6 +52,22 @@
     <item name="android:textColorPrimary">@color/component_color_shared_primary_dark_text_color</item>
   </style>
 
+  <style name="AlertDialogBackgroundPositiveButtonTheme" parent="Theme.MaterialComponents.Light.Dialog.Alert.Bridge">
```

Comment:
This implies that it is a style to be applied to the buttons specifically, and not to the alert dialog itself. Is that the case? In which case should it be in a styles file?